tests/mapping/test_CrossColumnMappingTransformer.py

```diff
@@ -214,3 +164,15 @@ def test_mappings_unchanged(self):
             actual=x.mappings,
             msg="CrossColumnMappingTransformer.transform has changed self.mappings unexpectedly",
         )
```

Comment:
This test is already inherited from BaseMappingTransformerTransformTests

Reply:
removed  [00a7535](https://github.com/lvgig/tubular/pull/192/commits/00a753509eac6bc1c14f7dc672c78f10d5a99523)

Reply:
thanks